Abhi-AnimeX/app/src/main/java/com/abhijith/animex/data/repository/ISeasonalAnimeListRepositoryImpl.kt

```diff
@@ -0,0 +1,29 @@
+package com.abhijith.animex.data.repository
+
+import com.abhijith.animex.data.network.AnimeXApiService
+import com.abhijith.animex.data.network.model.AnimeListResponse
+import com.abhijith.animex.domain.repository.ISeasonalAnimeListRepository
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.flow
+import kotlinx.coroutines.flow.flowOn
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@Singleton
```

Comment:
why?

Reply:
i had issues with hilt earlier. so was trying all possible scenarios to fix it. forgot to remove it